src/commands/generateProject.ts

```diff
@@ -84,14 +83,22 @@ export async function generateProject(clickedFileUri: vscode.Uri | undefined): P
       apiPackageName +
       " --model-package " +
       modelPackageName;
-    // run the generator command with a "progress" dialog
-    await vscode.window.withProgress(
-      {
-        location: vscode.ProgressLocation.Notification,
-        title: "Generating the MicroProfile Rest Client interface template...",
-      },
-      () => processUtil.exec(jarCommand)
-    );
+    try {
+      await generateRestClient(jarCommand);
+    } catch (e) {
+      if (e.message.includes(SPEC_VALIDATION_EXCEPTION)) {
+        const selection = await vscode.window.showInformationMessage(
```

Comment:
Could potentially use `showErrorMessage` here instead. Not sure which is more appropriate if spec validation fails.